odpf/optimus/core/v1beta1/job_run.proto

```diff
@@ -98,13 +105,32 @@ message RegisterInstanceRequest {
   string jobrun_id = 7;
 }
 
+message JobRunInputRequest {
+  reserved 3;
+
+  string project_name = 1;
+  string job_name = 2;
+  google.protobuf.Timestamp scheduled_at = 4;
+
+  string instance_name = 5;
+  InstanceSpec.Type instance_type = 6;
+
+  // either set job_name if this is a scheduled execution
+  // or set jonrun_id if this is a manual triggered execution
+  // and not really registered as a valid job
+  string jobrun_id = 7;
+}
+
 message RegisterInstanceResponse {
   ProjectSpecification project = 1;
   NamespaceSpecification namespace = 4;
   JobSpecification job = 2;
 
   InstanceSpec instance = 3;
-  InstanceContext context = 5;
+  
+  reserved 5;
```

Comment:
if it  is just a rename lets not bother about backward compatibility imo as all the existing clients will still work.